R/format_ay.R

```diff
@@ -32,7 +32,7 @@ format_ay <- function(year) {
 
 format_ay_reverse <- function(year) {
   if (!grepl("^\\d{4}/\\d{2}.*", year)) {
-    stop("year parameter must be a seven digit string e.g. '2016/17'")
+    stop("year parameter must in the format 2016/17")
```

Comment:
How about a blend of this and what we had before - "year parameter must a 7 digit string in an academic year format, e.g. '2017/18'"?